recipes-bsp/chip-u-boot-scr/chip-u-boot-scr.bb

```diff
@@ -0,0 +1,89 @@
+SUMMARY = "U-boot boot scripts for CHIP boards"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+COMPATIBLE_MACHINE = "chip"
```

Comment:
Usually the COMPATIBLE_MACHINE is at the end of the recipe: https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta-skeleton/recipes-kernel/linux/linux-yocto-custom.bb .